bundles/org.eclipse.swt/Eclipse SWT/gtk/org/eclipse/swt/widgets/List.java

```diff
@@ -1299,6 +1292,20 @@ public void removeSelectionListener(SelectionListener listener) {
 	eventTable.unhook (SWT.DefaultSelection,listener);
 }
 
+private boolean search_enabled() {
```

Comment:
`search_enabled` should be better named `searchEnabled` to be consistent with existing code in Tree/Table.

Reply:
=> see https://github.com/eclipse-platform/eclipse.platform.swt/pull/1934